Tests/SonarScanner.MSBuild.PreProcessor.Test/JreCaching/TarGzUnpackTests.cs

```diff
@@ -56,13 +59,15 @@ public void TarGzUnpacking_Success()
         using var archive = new MemoryStream(Convert.FromBase64String(sampleTarGzFile));
         using var unzipped = new MemoryStream();
         fileWrapper.Create($"""{baseDirectory}\Main\Sub2\Sample.txt""").Returns(unzipped);
+        filePermissionsWrapper.When(x => x.Copy(Arg.Any<TarEntry>(), Arg.Any<string>())).Throw(new Exception("Sample exception message"));
```

Comment:
The only relevant change as far as tests are concerned.

Reply:
Could you please update the test name to reflect the flow?
Unpacking -> Success, Copy -> fail